src/auslib/services/releases.py

```diff
@@ -490,6 +491,8 @@ def update_release(name, blob, old_data_versions, when, changed_by, trans):
         if current_assets.get(str_path):
             if item != current_assets[str_path]["data"]:
                 old_data_version = get_by_path(old_data_versions, path)
+                if not old_data_version:
+                    raise OutdatedDataError(f"Missing data_version for {path}")
```

Comment:
nit: `ValueError` might be more approciate here. At least to me, `OutdatedDataError` implies that one was sent, but was wrong.

Reply:
Updated, thanks.